cmd/liqoctl/cmd/offload.go

```diff
@@ -25,29 +25,38 @@ import (
 )
 
 func newOffloadCommand(ctx context.Context) *cobra.Command {
-	var offloadClusterCmd = &cobra.Command{
+	var offloadCommand = &cobra.Command{
 		Use:          offload.UseCommand,
 		SilenceUsage: true,
 		Short:        offload.LiqoctlOffloadShortHelp,
 		Long:         offload.LiqoctlOffloadLongHelp,
-		Args:         cobra.MinimumNArgs(2),
+	}
+	offloadCommand.AddCommand(newNamespaceCommand(ctx))
+	return offloadCommand
+}
+
+func newNamespaceCommand(ctx context.Context) *cobra.Command {
+	var offloadClusterCmd = &cobra.Command{
+		Use:          offload.ClusterResourceName,
+		SilenceUsage: true,
+		Short:        offload.LiqoctlOffloadShortHelp,
+		Long:         offload.LiqoctlOffloadLongHelp,
+		Args:         cobra.MinimumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return offload.HandleOffloadCommand(ctx, cmd, args)
 		},
 	}
 	podOffloadingStrategy := args.NewEnum([]string{string(offloadingv1alpha1.LocalAndRemotePodOffloadingStrategyType),
 		string(offloadingv1alpha1.RemotePodOffloadingStrategyType),
-		string(offloadingv1alpha1.LocalPodOffloadingStrategyType)},
-		string(offloadingv1alpha1.LocalAndRemotePodOffloadingStrategyType))
+		string(offloadingv1alpha1.LocalPodOffloadingStrategyType)}, string(offloadingv1alpha1.LocalAndRemotePodOffloadingStrategyType))
 
-	offloadClusterCmd.PersistentFlags().Var(podOffloadingStrategy,
-		offload.PodOffloadingStrategyFlag, offload.PodOffloadingStrategyHelp)
+	offloadClusterCmd.PersistentFlags().Var(podOffloadingStrategy, offload.PodOffloadingStrategyFlag, "Select the desired pod offloading strategy")
```

Comment:
Are these changes desired? Should we explain which are the supported values?

Reply:
Thanks for noticing.